.github/workflows/gha.yml

```diff
@@ -34,8 +35,9 @@ jobs:
         with:
           fetch-depth: 50
 
+      # "pull" or "rebuild" can be passed to a secret FORCE_IMAGE_ACTION to override default action
```

Comment:
why exactly is this change needed?

Reply:
when it happens (and we know from the past it did) that image is missing on the container registry, for some reason, we can easily set this secret using GH interface, and then easily rebuilt the image by re-running failed (master) build, using GHA interface. This way we don't need to make this dummy change with whitespace or something to force rebuild image.